.github/workflows/windows.yml

```diff
@@ -247,20 +246,21 @@ jobs:
       - name: CMake configure - tokenizers
         run: |
           cmake -DBUILD_FAST_TOKENIZERS="${{ matrix.build_fast_tokenizers }}" `
+                -DCMAKE_BUILD_TYPE="${{ matrix.build_type }}" `
```

Comment:
I suppose it's not required here, because you are using multi-config generator, right?

Reply:
removed